docs/en/FACL.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: global
+title: Access Control List
+nickname: Access Control List
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+This document describes the Access Control List feature related to file and directory permissions in Alluxio. 
+
+1. [ACL Permission Model](#ACL Permission Model): Alluxio implements an ACL model similar to those found in Linux and HDFS. This section describes some background on what this permission model looks like. 
+1. [Command Line Interface](#Command Line Interface): The users can interact with Alluxio's ACL model by using the command line interface tools such as `setfacl` and `getfacl`. 
+
+## ACL Permission Model
+Alluxio already implements a file and directory permission model similar to traditional Unix permission bits. For each directory and file, there are three types of permissions: read, write and execute. These permissions can be assigned to three different user clases: owner, group and other. 
+
+The permission bits model is sufficient for most cases. However, it is not flexible enough to express more complicated security policies found in today's computing environment. Access Control List (ACL) is a feature which allows users to mnanage any user or group's rights to any file system objects.
+
+In Alluxio's ACL model, a file or directory's ACL consists of many ACL entries. Each entry specifies a particular user or group's permission to read, write and execute. Each ACL entry consists of a type, which can be one of user, group or mask, an optional name and a permission string similar to the linux permission bits. For example, `user::rw-` is an ACL entry. This entry has the type `user`, with an unspecified name, which means the owner of the file. `rw-` means the owner of the file has `read` and `write` permissions but no `execute` permission. Another example is `group:interns:r--`, this means the interns group has read-only access to the file. 
```

Comment:
We usually use line breaks to prevent lines from wrapping too much. This is helpful for seeing diffs during review, and helps with reading/editing the docs from a terminal.